app/helpers/application_helper.rb

```diff
@@ -1,6 +1,6 @@
 module ApplicationHelper
   def full_title page_title = ""
-    base_title = t(base.title)
+    base_title = t "base.title"
```

Comment:
trong helper thì phải khai báo đầy đủ
```suggestion
    base_title = I18n.t "base.title"
```